docs/network/hotplug_interfaces.md

```diff
@@ -24,11 +26,6 @@ to be added to a running pod. This is not trivial, and has some requirements:
   this Multus version exposes an endpoint to create attachments for a given pod
   on demand.
 
-### Enabling network interface hotplug support
```

Comment:
We still support old releases that require this FG. Can you place it under a note so users with those versions will still have a ref?